Adding ballpark assignment

matthew-zahn opened the pull request.

matthew-zahn committed:
Updates

matthew-zahn committed:
updates

matthew-zahn committed:
update

matthew-zahn committed:
updated typos

llorracc commented:
Looks good - except for LaTeX problems; see if you can fix those (probably you have some extension installed).

matthew-zahn committed:
Update Zahn_WongMPTransimission.ipynb

Updated equations to possibly improve Github rendering on other computers. Other issues may also be browser that is used or operating system. Does not appear to be tied to a single Jupyter extention.
